src/ucp/proto/proto_am.inl

```diff
@@ -348,6 +348,9 @@ ucs_status_t ucp_do_am_zcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
         ucp_send_request_add_reg_lane(req, req->send.lane);
     } else {
         req->send.lane = ucp_ep_get_am_lane(ep);
+        if (req->send.state.dt.offset == 0) {
+            ucp_send_request_add_reg_lane(req, req->send.lane);
```

Comment:
maybe move this after line 354?
seems only case ucp_send_request_add_reg_lane is not called, is when enable_am_bw is false